From cf6dfc4323b972db52a4e72ce2f884197a10cc90 Mon Sep 17 00:00:00 2001
From: Raymond Wong <infiwang@pm.me>
Date: Wed, 22 Feb 2023 23:16:44 +0800
Subject: [PATCH 056/156] riscv(asm): drop unused irl in bnot

NOT a,b = XORI a,b,0xfff, therefore bnot should not have any fuse oppotunity.

This is a WIP, archive only.
---
 src/lj_asm_riscv64.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/lj_asm_riscv64.h b/src/lj_asm_riscv64.h
index 8d6c31b2..da7c4930 100644
--- a/src/lj_asm_riscv64.h
+++ b/src/lj_asm_riscv64.h
@@ -1362,7 +1362,6 @@ static void asm_mulov(ASMState *as, IRIns *ir)
 static void asm_bnot(ASMState *as, IRIns *ir)
 {
   Reg left, dest = ra_dest(as, ir, RSET_GPR);
-  IRIns *irl = IR(ir->op1);
   left = ra_hintalloc(as, ir->op1, dest, RSET_GPR);
   emit_ds(as, RISCVI_NOT, dest, left);
 }
-- 
2.42.0

